sdk/python/foundation-models/system/inference/fill-mask/book-corpus-dataset/download-dataset.py

```diff
@@ -0,0 +1,32 @@
+# import library to parse command line arguments
+import argparse, os, json
+
+parser = argparse.ArgumentParser()
+# add an argument to specify a dataset name to download
+parser.add_argument("--dataset", type=str, default="bookcorpus", help="dataset name")
+# add an argument to specify the directory to download the dataset to
+parser.add_argument(
+    "--download_dir",
+    type=str,
+    default="./",
+    help="directory to download the dataset to",
+)
+args = parser.parse_args()
+
+# create the download directory if it does not exist
+if not os.path.exists(args.download_dir):
+    os.makedirs(args.download_dir)
+
+# import hugging face datasets library
+from datasets import load_dataset, get_dataset_split_names
+
+for split in get_dataset_split_names(args.dataset):
```

Comment:
Do you need it for every split? Why not just "train"?

Reply:
Updated to just download "train"